blueprints/cloud-operations/network-quota-monitoring/src/plugins/utils.py

```diff
@@ -34,7 +34,7 @@
 Host: compute.googleapis.com
 
 '''
-RE_URL = re.compile(r'nextPageToken=[^&]+&?')
+RE_URL = re.compile(r'pageToken=[^&]+&?')
```

Comment:
I forgot one last change here, in the body of the HTTP reply it is called 'nextPageToken'  but in the URL it should be 'pageToken'.